app/writer.py

```diff
@@ -45,6 +49,24 @@ def write_calendar(fname, cal):
     with open(cal_path, 'wb') as cal_file:
         cal_file.write(cal.to_ical(True))
 
+def get_calendar(uuid_maker=None):
+    MENU_FILE = 'menu.ics'
```

Comment:
These are all defined here and at the top.  Only need 1 of them.  I prefer top since they are constants.